scripts/lint_prowjobs/main.go

```diff
@@ -0,0 +1,290 @@
+	return presubmitCheck(func(presubmitConfig config.Presubmit, fileContentsString string) (bool, string, string) {
+		if presubmitConfig.JobBase.UtilityConfig.DecorationConfig.GCSConfiguration.Bucket != jc.Bucket {
+			return false, findLineNumber(fileContentsString, "bucket:"), fmt.Sprintf("Incorrect bucket configuration, please configure S3 bucket as => bucket: %s", jc.Bucket)
+		}
+		return true, "", ""
+	})
+}
+
+func PostsubmitBucketCheck(jc *JobConstants) postsubmitCheck {
+	return postsubmitCheck(func(postsubmitConfig config.Postsubmit, fileContentsString string) (bool, string, string) {
+		if postsubmitConfig.JobBase.UtilityConfig.DecorationConfig.GCSConfiguration.Bucket != jc.Bucket {
+			return false, findLineNumber(fileContentsString, "bucket:"), fmt.Sprintf("Incorrect bucket configuration, please configure S3 bucket as => bucket: %s", jc.Bucket)
+		}
+		return true, "", ""
+	})
+}
+
+func PresubmitClusterCheck(jc *JobConstants) presubmitCheck {
+	return presubmitCheck(func(presubmitConfig config.Presubmit, fileContentsString string) (bool, string, string) {
+		if presubmitConfig.JobBase.Cluster != jc.Cluster {
+			return false, findLineNumber(fileContentsString, "cluster:"), fmt.Sprintf("Incorrect cluster configuration, please configure cluster as => cluster: %s", jc.Cluster)
+		}
+		return true, "", ""
+	})
+}
+
+func PostsubmitClusterCheck(jc *JobConstants) postsubmitCheck {
+	return postsubmitCheck(func(postsubmitConfig config.Postsubmit, fileContentsString string) (bool, string, string) {
+		if postsubmitConfig.JobBase.Cluster != jc.Cluster {
+			return false, findLineNumber(fileContentsString, "cluster:"), fmt.Sprintf("Incorrect cluster configuration, please configure cluster as => cluster: %s", jc.Cluster)
+		}
+		return true, "", ""
+	})
+}
+
+func ServiceAccountCheck(jc *JobConstants) presubmitCheck {
+	return presubmitCheck(func(presubmitConfig config.Presubmit, fileContentsString string) (bool, string, string) {
+		if presubmitConfig.JobBase.Spec.ServiceAccountName != jc.ServiceAccountName {
+			return false, findLineNumber(fileContentsString, "skip_report:"), fmt.Sprintf("Incorrect service account configuration, please configure service account as => serviceaccountName: %s", jc.ServiceAccountName)
+		}
+		return true, "", ""
+	})
+}
+
+func PresubmitMakeTargetCheck(jc *JobConstants) presubmitCheck {
+	return presubmitCheck(func(presubmitConfig config.Presubmit, fileContentsString string) (bool, string, string) {
+		jobMakeTargetMatches := regexp.MustCompile(`make (\w+) .*`).FindStringSubmatch(strings.Join(presubmitConfig.JobBase.Spec.Containers[0].Command, " "))
+		jobMakeTarget := jobMakeTargetMatches[len(jobMakeTargetMatches)-1]
+		makeCommandLineNo := findLineNumber(fileContentsString, "make")
+		if strings.Contains(presubmitConfig.JobBase.Name, "cni") && jobMakeTarget != jc.CniMakeTarget {
+			return false, makeCommandLineNo, fmt.Sprintf("Invalid make target, please use the \"%s\" target", jc.CniMakeTarget)
```

Comment:
nit: use backticks for clarity, same for others
```suggestion
			return false, makeCommandLineNo, fmt.Sprintf(`Invalid make target, please use the "%s" target`, jc.CniMakeTarget)
```